database/sql/create_table-test_failures.sql

```diff
@@ -3,5 +3,6 @@ CREATE TABLE IF NOT EXISTS test_failures(
     error_name TEXT NOT NULL,
     package_name TEXT NOT NULL,
     job_name TEXT NOT NULL,
-    build_number INT NOT NULL
-);
\ No newline at end of file
+    build_number INT NOT NULL,
+    age INT NOT NULL
```

Comment:
How do we plan to calculate age here? I'm wondering if this will degrade the time that the daily report takes to run.

Reply:
Actually, the age is a static value retrieved by jenkins API 

Example: https://ci.ros2.org/view/nightly/job/nightly_win_deb/3134/testReport/api/xml?tree=suites[name,cases[age,name,status]] (search for `<age>2</age>`)

Reply:
We discussed this on standup. My main concern is that the age retrieved by jenkins, might differ from the <<real age>> of an error for flaky tests and when there are build regressions.

However, we agreed that for now just retrieving it from jenkins would be a low hanging fruit that may give us benefits on the long term.